lib/facter/resolvers/aix/mountpoints.rb

```diff
@@ -31,9 +31,14 @@ def read_mount(fact_name)
           def add_mount_points_fact(line)
             elem = line.split("\s")
 
-            elem.shift unless line[0] == ' '
+            if line[0] != ' '
+              server = elem.shift
+              device = "#{server}:#{elem[0]}"
+            else
+              device = elem[0]
+            end
```

Comment:
Ah interesting, there was a previous fix 6c6ef2df386c416e65914048e0ad4749da78b8a1 to just ignore the `server`. Could you mention that in your commit message?